<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>URLList</class>
 <widget class="QMainWindow" name="URLList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>选择下载</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>解析 BVID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="bvid_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="video_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>530</width>
      <height>310</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="download_btn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>下载选中的视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_download_path_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>设置下载目录</string>
    </property>
   </widget>
   <widget class="QLabel" name="download_path">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
